<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebuggerWindow</class>
 <widget class="QMainWindow" name="DebuggerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpoLua Debugger</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QStackedWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1077</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionPause"/>
    <addaction name="actionContinue"/>
    <addaction name="actionStop"/>
    <addaction name="actionRestart"/>
    <addaction name="separator"/>
    <addaction name="actionStepOver"/>
    <addaction name="actionStepInto"/>
    <addaction name="actionStepOut"/>
    <addaction name="actionToggleBreak"/>
    <addaction name="separator"/>
    <addaction name="breakOnError"/>
    <addaction name="windowFocusEnabled"/>
    <addaction name="actionFlush"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="variablesDockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFeature::DockWidgetFloatable|QDockWidget::DockWidgetFeature::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Stack Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="StackView" name="stackView">
    <attribute name="headerMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="headerDefaultSectionSize">
     <number>140</number>
    </attribute>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPause"/>
   <addaction name="actionContinue"/>
   <addaction name="actionStop"/>
   <addaction name="actionRestart"/>
   <addaction name="actionStepOver"/>
   <addaction name="actionStepInto"/>
   <addaction name="actionStepOut"/>
  </widget>
  <widget class="QDockWidget" name="modulesDockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFeature::DockWidgetClosable|QDockWidget::DockWidgetFeature::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Loaded Modules</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QTreeWidget" name="modulesView">
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="headerMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="headerDefaultSectionSize">
     <number>200</number>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">2</string>
     </property>
    </column>
   </widget>
  </widget>
  <action name="actionPause">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackPause"/>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackStop"/>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Shift+F5</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionRestart">
   <property name="icon">
    <iconset theme="view-refresh"/>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+R</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionContinue">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionStepInto">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::GoDown"/>
   </property>
   <property name="text">
    <string>Step Into</string>
   </property>
   <property name="toolTip">
    <string>Step Into</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionStepOver">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::GoNext"/>
   </property>
   <property name="text">
    <string>Step Over</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionStepOut">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::GoUp"/>
   </property>
   <property name="text">
    <string>Step Out</string>
   </property>
   <property name="shortcut">
    <string>Shift+F11</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="breakOnError">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Break on Error</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="windowFocusEnabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Send Window Focus Events</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionToggleBreak">
   <property name="text">
    <string>Toggle Breakpoint</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFlush">
   <property name="text">
    <string>Flush Graphics Operations</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StackView</class>
   <extends>QTreeView</extends>
   <header>stackview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
